File: Cafebook/CafebookModel/Model/ModelWeb/KhachHang/LienHeDto.cs
namespace CafebookModel.Model.ModelWeb.KhachHang
{
    public class LienHeDto
    {
        public string? TenQuan { get; set; }
        public string? GioiThieu { get; set; }
        public string? DiaChi { get; set; }
        public string? SoDienThoai { get; set; }
        public string? EmailLienHe { get; set; }
        public string? GioHoatDong { get; set; }
        public string? LinkFacebook { get; set; }
        public string? LinkInstagram { get; set; }
        public string? LinkGoogleMapsEmbed { get; set; }
        public string? LinkZalo { get; set; }
        public string? LinkYoutube { get; set; }
        public string? LinkWebsite { get; set; }
        public string? LinkX { get; set; }
    }

    public class PhanHoiInputModel
    {
        public string Ten { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string NoiDung { get; set; } = string.Empty;
    }
}

File: Cafebook/ADRCafebookApi/app/src/main/java/com/example/cafebook/models/LienHeDto.java
package com.example.cafebook.models;

import com.google.gson.annotations.SerializedName;

public class LienHeDto {
    @SerializedName("tenQuan") public String tenQuan;
    @SerializedName("gioiThieu") public String gioiThieu;
    @SerializedName("diaChi") public String diaChi;
    @SerializedName("soDienThoai") public String soDienThoai;
    @SerializedName("emailLienHe") public String emailLienHe;
    @SerializedName("gioHoatDong") public String gioHoatDong;
    @SerializedName("linkFacebook") public String linkFacebook;
    @SerializedName("linkZalo") public String linkZalo;
}
